loony_dev/web/static/ds/ld-system.css

```
@@ -0,0 +1,353 @@
+table.board { border-collapse: collapse; width: 100%; }
+table.board th { text-align: left; font-size: 10.5px; letter-spacing: .12em; text-transform: uppercase; color: var(--fg-muted); font-weight: 700; padding: 12px 14px; border-bottom: 1px solid var(--border-default); }
+table.board td { padding: 12px 14px; border-bottom: 1px solid var(--border-hairline); vertical-align: middle; font-size: 13.5px; }
+table.board tbody tr { cursor: pointer; }
+table.board tbody tr:hover td { background: var(--bg-subtle); }
+table.board tbody tr:last-child td { border-bottom: 0; }
+/* Container-query responsive board: when the .board-wrap CONTAINER is narrow,
+   the table reflows to stacked cards (label via data-label) instead of
+   overflowing — native CSS, no JS, no page-level media-query breakpoints. */
+.board-wrap { container-type: inline-size; overflow-x: auto; }
+@container (max-width: 600px) {
+  .board-wrap table.board thead { position: absolute; width: 1px; height: 1px; overflow: hidden; clip: rect(0 0 0 0); }
```

Stylelint (17.13.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)